.github/ISSUE_TEMPLATE/citus-enterprise-release-checklist-template.md

```diff
@@ -0,0 +1,55 @@
+---
+name: citus-enterprise release checklist template
+about: Please make sure to follow the checklist below along the release process.
+title: Citus Enterprise release checklist - [vX.Y.Z]
+labels: release_checklist
+assignees: gurkanindibay
+
+---
+
+# Update OS Packages
+## Debian and RedHat
+- Change your directory to `packaging` repository directory & checkout `all-enterprise` branch.
+- [ ] Run [this pipeline](https://github.com/citusdata/packaging/actions/workflows/update_package_properties.yml) using branch name as `all-enterprise`. Input tag name and if version is fancy, input the fancy version number. Other parameters could be kept as is if you want.
+  - Then check the following (needed for both debian & redhat):
+    - [ ] Updated `pkglatest` variable in the `pkgvars` file to `$VERSION.citus-1`
+  - Then check the following (needed for debian):
+    - [ ] A new entry (`$VERSION.citus-1`, `stable`) is added to the `debian/changelog` file
+  - Then check the following (needed for redhat):
+    - [ ] `citus.spec` file is updated:
+      - [ ] `Version:` field
+      - [ ] `Source0:` field
+      - [ ] A new entry (`$VERSION.citus-1`) in the `%changelog` section
+- [ ] Get changes reviewed; merge the PR
+- [ ] Ensure Github Actions builds completed successfully and package count for each os is as below table and packages in postgres versions is compliant with `postgres-matrix.yml` in the `all-project` branch
+
+https://github.com/citusdata/tools/blob/be12af3b8f435d17a52e607c666f6b15379f5970/packaging_automation/tests/test_citus_package.py#L21-L33
```

Comment:
What about having a human readable link with some explanation that will make it easier for the reader to understand why this is here?

If there is no reason to have it here, you can drop this line as well.

Reply:
- [ ] Ensure Github Actions builds completed successfully and package count for each os is as below table and packages in postgres versions is compliant with `postgres-matrix.yml` in the `all-project` branch

This is the description but it seems that it is not as definitive as I intended
What's your suggestion?

Reply:
There are some issues with the current link:

1. The link contains a sha, and that means the link needs to be updated when new distros are introduced in the file or the package counts for a distro changed for some reason.
2. The tests are supposed to check the number of generated files, right? It does not make sense for an issue template to have an item that wants the user to read some piece of code, and use the values in there to check if the count of generated packages are correct.

My suggestion is to either find a better and easier alternative to this, or drop the link completely.

A possible alternative is to check the total number of packages generated, and dive deeper if the total does not match.